be-elecprice/crawler/jnb.go

```
@@ -0,0 +1,332 @@
+	req.Header.Set("Referer", "https://jnb.ccnu.edu.cn/MobilePayWeb_Vue/")
+	req.Header.Set("Origin", "https://jnb.ccnu.edu.cn")
+	req.Header.Set("User-Agent", "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/131.0.0.0 Safari/537.36")
+	if token != "" {
+		req.Header.Set("Authorization", "Bearer "+token)
+	}
+
+	resp, err := c.pc.NewProxyClient(proxy.WithProxyTransport()).Do(req)
+	if err != nil {
+		return 0, "", fmt.Errorf("发送请求失败: %w", err)
+	}
+	defer resp.Body.Close()
```

golangci-lint (2.12.2)

[error] Error return value of `resp.Body.Close` is not checked

(errcheck)